src/System.Net.Ping/src/System/Net/NetworkInformation/Ping.Unix.cs

```diff
@@ -175,9 +175,9 @@ public partial class Ping
             else
             {
                 cts.Cancel();
-                if (p.ExitCode != 0)
+                if (p.ExitCode == 1 || p.ExitCode == 2)
```

Comment:
https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man8/ping6.8.html, it's from osx docs. On linux, man ping, man ping6.. Previously i put doc link in code, and someone had mentioned not to do that, hence not putting any external doc links in code from then.